### PR TITLE
Handle BIPM and wideband DMs correctly in `make_fake_toas_fromtim` 

### PR DESCRIPTION
**Pull request by abhisrkckl**

### CODE REVIEW

File: src/pint/simulation.py [hunk 1]
```diff
@@ -514,25 +514,14 @@ def make_fake_toas_fromtim(
...
-        if hasattr(model, "PLANET_SHAPIRO") and model.PLANET_SHAPIRO.value is not None
-        else False
-    )
-
-    input_ts = pint.toa.get_TOAs(timfile, ephem=ephem, planets=planets)
+    input_ts = pint.toa.get_TOAs(timfile, model=model)
 
     if input_ts.is_wideband():
         dm_errors = input_ts.get_dm_errors()
         ts = update_fake_dms(model, input_ts, dm_errors, add_noise)
```
Comment (dlakaplan) [inline src/pint/simulation.py:521]: You are creating `ts` here if it's Wideband, but if it's not Wideband then I don't know if that variable is used.  which makes the command below suspect.

Comment (abhisrkckl) [reply src/pint/simulation.py]: Corrected.

**Comment by abhisrkckl** [comment]
This is ready to go.

File: src/pint/simulation.py [hunk 2]
```diff
@@ -514,25 +514,14 @@ def make_fake_toas_fromtim(
...
-        model.EPHEM.value
-        if hasattr(model, "EPHEM") and model.EPHEM.value is not None
-        else None
-    )
-    planets = (
-        model.PLANET_SHAPIRO.value
-        if hasattr(model, "PLANET_SHAPIRO") and model.PLANET_SHAPIRO.value is not None
-        else False
-    )
+    ts = pint.toa.get_TOAs(timfile, model=model)
```
Comment (dlakaplan) [inline src/pint/simulation.py:517]: So setting the model attribute will propagate BIPM, planets, etc, right?

Comment (abhisrkckl) [reply src/pint/simulation.py:517]: It should.

**Pull request merged**